data/json/recipes/recipe_weapon.json

```diff
@@ -852,6 +852,35 @@
     "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
     "components": [ [ [ "pipe", 2 ] ], [ [ "scrap", 3 ] ], [ [ "2x4", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "lever_shotgun",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ "gun", 3 ],
+    "difficulty": 7,
+    "time": "3 h",
+    "reversible": true,
+    "autolearn": true,
```

Comment:
This (and the other recipes) probably shouldn't be autolearned, but something you can pick up from a relevant book.